ansible/roles/hana_checks/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: HANA check - Wait pacemaker up
+  become: true
+  command: systemctl --no-pager is-active pacemaker
+  register: pm
+  retries: "{{ pacemaker_timeout // 15 }}"
+  delay: 15
+  until: pm.stdout == 'active'
+
+- name: HANA check - Wait cluster idle again
+  become: true
+  command: cs_wait_for_idle --sleep 5
+  register: idle2
+  retries: "{{ cs_wait_timeout // 5 }}"
+  delay: 5
+  until: idle2.rc == 0
+
+- name: HANA check - Gather topology JSON
```

Comment:
here you say JSON ...

Reply:
ah, yes - this didn't work and I switched to script. Let me fix